meta-resin-common/recipes-core/dropbear/files/atomic-hostkey.patch

```diff
@@ -0,0 +1,44 @@
+Apply https://github.com/mkj/dropbear/pull/36 while waiting for it to be
```

Comment:
It's better to have a fraction of the patch and have this mention in the commit log.